examples/verif-manual/vm-291.py

```diff
@@ -0,0 +1,574 @@
+
+    # Enter POST1 module (Post-processing processor)
+    mapdl.post1()
+    # Set the current results set to the last set to be read from result file
+    mapdl.set("LAST")
+    # redirects output to the default system output file
+    mapdl.run("/OUT")
+    # reactivates suppressed printout
+    mapdl.run("/GOPR")
+
+# Set constant parameters, R1=1 and Z1=1
+mapdl.run("R1=1")
+mapdl.run("Z1=1")
+
+# UY AT NODE (1,0,0)
+mapdl.run("UY1=P*(1-NUxy**2)/(PI*Exx*R1)               ")
+# UY AT NODE (0,1,0)
+mapdl.run("UP1=P/(2*PI*Exx*Z1)*(1+NUxy+2-2*NUxy**2)    ")
+# MAPDL UY AT NODE(1,0,0)
+mapdl.get("UYA1", "NODE", 2, "U", "Y")
+# MADPL UY AT NODE(0,1,0)
+mapdl.get("UPA1", "NODE", 4, "U", "Y")
+
+# Set constant parameters, R2=2 and Z2=2
+mapdl.run("R2=2")
+mapdl.run("Z2=2")
+
+# UY AT NODE (2,0,0)
+mapdl.run("UY2=P*(1-NUxy**2)/(PI*Exx*R2)               ")
+# UY AT NODE (0,2,0)
+mapdl.run("UP2=P/(2*PI*Exx*Z2)*(1+NUxy+2-2*NUxy**2)    ")
+# MAPDL UY AT NODE(2,0,0)
+mapdl.get("UYA2", "NODE", 5, "U", "Y")
+# MADPL UY AT NODE(0,2,0)
+mapdl.get("UPA2", "NODE", 9, "U", "Y")
+
+# Set constant parameters, R3=3 and Z3=3
+mapdl.run("R3=3")
+mapdl.run("Z3=3")
+
+# UY AT NODE (3,0,0)
+mapdl.run("UY3=P*(1-NUxy**2)/(PI*Exx*R3)               ")
+# UY AT NODE (0,3,0)
+mapdl.run("UP3=P/(2*PI*Exx*Z3)*(1+NUxy+2-2*NUxy**2)    ")
+# MAPDL UY AT NODE(3,0,0)
+mapdl.get("UYA3", "NODE", 10, "U", "Y")
+# MADPL UY AT NODE(0,3,0)
+mapdl.get("UPA3", "NODE", 14, "U", "Y")
+
+# Set constant parameters, R4=4 and Z4=4
+mapdl.run("R4=4")
+mapdl.run("Z4=4")
+
+# UY AT NODE (4,0,0)
+mapdl.run("UY4=P*(1-NUxy**2)/(PI*Exx*R4)               ")
+# UY AT NODE (0,4,0)
+mapdl.run("UP4=P/(2*PI*Exx*Z4)*(1+NUxy+2-2*NUxy**2)    ")
+# MAPDL UY AT NODE(4,0,0)
+mapdl.get("UYA4", "NODE", 15, "U", "Y")
+# MADPL UY AT NODE(0,4,0)
+mapdl.get("UPA4", "NODE", 19, "U", "Y")
+
+# Enter non-interactive mode
+with mapdl.non_interactive:
+    # redirects output to the default system output file
+    mapdl.run("/OUT,SCRATCH")
+
+    # Define dimensions for output
+    mapdl.dim("VALUE", "", 3, 6)
+    mapdl.dim("LABEL", "CHAR", 3, 2)
+    # Define labels for output
+    mapdl.run("VALUE(1,1)=UY2")
+    mapdl.run("VALUE(2,1)=UY3")
+    mapdl.run("VALUE(3,1)=UY4")
+    mapdl.run("VALUE(1,2)=UYA2")
+    mapdl.run("VALUE(2,2)=UYA3")
+    mapdl.run("VALUE(3,2)=UYA4")
+    mapdl.run("VALUE(1,3)=UP2")
+    mapdl.run("VALUE(2,3)=UP3")
+    mapdl.run("VALUE(3,3)=UP4")
+    mapdl.run("VALUE(1,4)=UPA2")
+    mapdl.run("VALUE(2,4)=UPA3")
+    mapdl.run("VALUE(3,4)=UPA4")
+    mapdl.run("VALUE(1,5)=UY2/UYA2")
+    mapdl.run("VALUE(2,5)=UY3/UYA3")
+    mapdl.run("VALUE(3,5)=UY4/UYA4")
+    mapdl.run("VALUE(1,6)=UP2/UPA2")
+    mapdl.run("VALUE(2,6)=UP3/UPA3")
+    mapdl.run("VALUE(3,6)=UP4/UPA4")
+    mapdl.run("LABEL(1,1)='NODE5'")
+    mapdl.run("LABEL(2,1)='NODE10'")
+    mapdl.run("LABEL(3,1)='NODE15'")
+    mapdl.run("LABEL(1,2)='NODE9'")
+    mapdl.run("LABEL(2,2)='NODE14'")
+    mapdl.run("LABEL(3,2)='NODE19'")
```

Comment:
Python should be used here, it demonstrates the flexibility and the communication between Python and PyMAPDL.

```suggestion
# Set constant parameters
r1=1
z1=1

# UY AT NODE (1,0,0)
uy1=p*(1-nuxy**2)/(pi*exx*r1)
# UY AT NODE (0,1,0)
up1=p/(2*pi*exx*z1)*(1+nuxy+2-2*nuxy**2)
# MAPDL UY AT NODE(1,0,0)
uya1 = mapdl.get("UYA1", "NODE", 2, "U", "Y")
# MADPL UY AT NODE(0,1,0)
upa1 = mapdl.get("UPA1", "NODE", 4, "U", "Y")

# Set constant parameters
r2=2
z2=2

# UY AT NODE (2,0,0)
uy2=p*(1-nuxy**2)/(pi*exx*r2)
# UY AT NODE (0,2,0)
up2=p*(2*pi*exx*z2)*(1+nuxy+2-2*nuxy**2)
# MAPDL UY AT NODE(2,0,0)
uya2 = mapdl.get("UYA2", "NODE", 5, "U", "Y")
# MADPL UY AT NODE(0,2,0)
upa2 = mapdl.get("UPA2", "NODE", 9, "U", "Y")

# Set constant parameters, R3=3 and Z3=3
r3=3
z3=3

# UY AT NODE (3,0,0)
uy3=p*(1-nuxy**2)/(pi*exx*r3)
# UY AT NODE (0,3,0)
up3=p/(2*pi*exx*z3)*(1+nuxy+2-2*nuxy**2)
# MAPDL UY AT NODE(3,0,0)
uya3 = mapdl.get("UYA3", "NODE", 10, "U", "Y")
# MADPL UY AT NODE(0,3,0)
upa3 = mapdl.get("UPA3", "NODE", 14, "U", "Y")

# Set constant parameters, R4=4 and Z4=4
r4 = 4
z4 = 4

# UY AT NODE (4,0,0)
uy4=p*(1-nuxy**2)/(pi*exx*r4)
# UY AT NODE (0,4,0)
up4=p/(2*pi*exx*z4)*(1+nuxy+2-2*nuxy**2)
# MAPDL UY AT NODE(4,0,0)
uya4 = mapdl.get("UYA4", "NODE", 15, "U", "Y")
# MADPL UY AT NODE(0,4,0)
upa4 = mapdl.get("UPA4", "NODE", 19, "U", "Y")

# Enter non-interactive mode
with mapdl.non_interactive:
    # redirects output to the default system output file
    mapdl.run("/OUT,SCRATCH")

    # Define dimensions for output
    mapdl.dim("VALUE", "", 3, 6)
    mapdl.dim("LABEL", "CHAR", 3, 2)
    # Define labels for output
    mapdl.run("VALUE(1,1)=UY2")
    mapdl.run("VALUE(2,1)=UY3")
    mapdl.run("VALUE(3,1)=UY4")
    mapdl.run("VALUE(1,2)=UYA2")
    mapdl.run("VALUE(2,2)=UYA3")
    mapdl.run("VALUE(3,2)=UYA4")
    mapdl.run("VALUE(1,3)=UP2")
    mapdl.run("VALUE(2,3)=UP3")
    mapdl.run("VALUE(3,3)=UP4")
    mapdl.run("VALUE(1,4)=UPA2")
    mapdl.run("VALUE(2,4)=UPA3")
    mapdl.run("VALUE(3,4)=UPA4")
    mapdl.run("VALUE(1,5)=UY2/UYA2")
    mapdl.run("VALUE(2,5)=UY3/UYA3")
    mapdl.run("VALUE(3,5)=UY4/UYA4")
    mapdl.run("VALUE(1,6)=UP2/UPA2")
    mapdl.run("VALUE(2,6)=UP3/UPA3")
    mapdl.run("VALUE(3,6)=UP4/UPA4")
    mapdl.run("LABEL(1,1)='NODE5'")
    mapdl.run("LABEL(2,1)='NODE10'")
    mapdl.run("LABEL(3,1)='NODE15'")
    mapdl.run("LABEL(1,2)='NODE9'")
    mapdl.run("LABEL(2,2)='NODE14'")
    mapdl.run("LABEL(3,2)='NODE19'")
```